<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addUserPage</class>
 <widget class="QDialog" name="addUserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200000</width>
    <height>200000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lbLogin">
         <property name="text">
          <string>Логин:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbPassword">
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbFullName">
         <property name="text">
          <string>Полное имя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbRole">
         <property name="text">
          <string>Роль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbClassId">
         <property name="text">
          <string>Идентификатор класса:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="leLogin"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lePassword"/>
       </item>
       <item>
        <widget class="QLineEdit" name="leFullName"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rbAdmin">
           <property name="text">
            <string>Администратор</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbTeatcher">
           <property name="text">
            <string>Преподаватель</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rbPupil">
           <property name="text">
            <string>Ученик</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="leClassId">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbCancelUser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddUser">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
